assets/index.html

```diff
@@ -21,46 +21,9 @@
   <title>evcc</title>
 
   <!-- CSS -->
-  <link href="css/bootstrap.min.css" rel="stylesheet">
-  <link href="css/fontawesome.min.css" rel="stylesheet">
-
-  <!-- Bootstrap -->
-  <script src="js/jquery-3.4.1.slim.min.js"></script>
-  <script src="js/popper.min.js"></script>
-  <script src="js/bootstrap.bundle.min.js"></script>
-
-  <!-- Axios -->
-  <script src="js/axios.min.js"></script>
-
-  <!-- VueJS -->
-  <script src="js/vue.min.js"></script>
-  <script src="js/vue-router.min.js"></script>
-
-  <style>
-    /* reverse loading animation */
-    .progress-bar-animated {
-      animation-direction: reverse;
-    }
-    .btn.caption {
-      opacity: 1;
-    }
-    .btn.first {
-      border-top-left-radius: .2rem;
-      border-bottom-left-radius: .2rem;
-    }
-    code {
-      font-size: 87.5%;
-      color: #e83e8c;
-      word-wrap: break-word;
-    }
-    .value, input[type=radio], label.btn {
-      white-space: nowrap !important;
-    }
-    .text-muted a, .text-muted a:hover {
-      color: #6c757d!important;
-      text-decoration: none;
-    }
-  </style>
+  <link href="../node_modules/bootstrap/dist/css/bootstrap.css" rel="stylesheet">
+  <link href="../node_modules/@fortawesome/fontawesome-free/css/all.css" rel="stylesheet">
```

Comment:
das kann eigentlich nicht stimmen- `fortawesome`?

Reply:
Hat mich auch irritiert, aber das scheint die offizielle NPM Organisation zu sein. Siehe https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers